lib/dor/workflow_exception.rb

```diff
@@ -0,0 +1,4 @@
+module Dor
+  class WorkflowException < ::RuntimeError
+  end
+end
```

Comment:
:ok_hand: although defer to you, @atz or @drh-stanford for commentary and evaluation on the implications for downstream projects (although I might have to take a look at SDR-PC robots).